types/office-js-preview/index.d.ts

```diff
@@ -16902,9 +16902,18 @@ declare namespace Office {
          */
         owner: String;
         /**
-         * The remote REST URL related to the owner’s mailbox.
+         * The REST API's base URL (currently https://outlook.office.com/api).
+         * Use with targetMailbox to construct REST operation's URL.
+         * 
+         * Example usage: `targetRestUrl + "/{api_version}/users/" + targetMailbox + "/{REST_operation}"`
+         */
+        targetRestUrl: String;
+        /**
+         * The target/owner's mailbox. Use with targetRestUrl to construct REST operation's URL.
+         * 
+         * Example usage: `targetRestUrl + "/{api_version}/users/" + targetMailbox + "/{REST_operation}"`
          */
-        restUrl: String;
```

Comment:
Seems like this reflects a breaking change—does the version number need to be bumped?

Reply:
Will discuss with my team today and get back to you.

Reply:
The version is just supposed to reflect the npm version of the JS library. So if this property was removed from the JS in a version bump, the version of the type defs should match that. If it was incorrectly there to begin with, the version string need not be updated.

Reply:
This is a preview API which is being refined so we think this is ok.